### PR TITLE
Filecoin-flavored Ganache blog post

### PR DESCRIPTION
**Pull request by seesemichaelj**
This PR is for a blog post requested by @Lizdaldal. There is an [internal Slack message](https://trufflesuite.slack.com/archives/C014QAVA0KA/p1617053458106600) with a link to a google docs link which you can review/suggest there rather than here in Github

### CODE REVIEW

File: src/blog/ganache-now-comes-with-the-filecoin-flavor.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Head over to the [documentation](/docs/filecoin/ganache/overview) to see Get Started guides whether you're using the NodeJS library, CLI, or graphical UI version of Ganache.
+
+Need an example on how to build a Filecoin application? Check out the [Filecoin Truffle Box](https://github.com/truffle-box/filecoin-box) to see an example to mint NFTs and provide a decentralized art gallery application.
+
+## Where can I get help?
+
+We're extremely excited to be working with the Filecoin team to add support for Filecoin into Truffle's tooling. If you have questions about our Filecoin integration or our plan for the future, join our Discord community and get your questions answered. We're happy to help!
+
+http://trfl.co/truffle-community
```
Comment (davidmurdoch) [inline src/blog/ganache-now-comes-with-the-filecoin-flavor.md:43]:
```suggestion
https://trfl.co/truffle-community
```

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

**Review by davidmurdoch — APPROVED** [review]

**Comment by seesemichaelj** [comment]
@DiscRiskandBisque, **do not** merge this one yet; we're still working on this one

**Comment by seesemichaelj** [comment]
@DiscRiskandBisque this is now ready!

**Review by kevinbluer — APPROVED** [review]
👍

**Pull request merged**